libbeat/docs/shared-brew-install.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[brew]]
+*brew:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {stack-version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+-------------------------
+brew tap elastic/tap
+brew install elastic/tap/{beatname_lc}@{brew-version}
```

Comment:
Specifying the version does not work for me. Not sure if it's the state of the current files, or if I'm doing something wrong.

Reply:
We're not going to document specifying a version. You now need to append `-full` to install the latest version of the default distribution.